src/nets/e2e_tts_th.py

```diff
@@ -30,27 +30,25 @@ def decoder_init(m):
         torch.nn.init.xavier_uniform_(m.weight, torch.nn.init.calculate_gain('tanh'))
 
 
-def make_mask(lengths, dim=None):
-    """FUNCTION TO MAKE BINARY MASK
+def make_non_pad_mask(lengths):
+    """Function to make mask tensor containing indices of non-padded part
 
-    Args:
-        length (list): list of lengths
-        dim (int): # dimension
+    e.g.: lengths = [5, 3, 2]
+          mask = [[1, 1, 1, 1 ,1],
+                  [1, 1, 1, 0, 0],
+                  [1, 1, 0, 0, 0]]
 
-    Return:
-        (torch.ByteTensor) binary mask tensor (B, Lmax, dim)
+    :param list lengths: list of lengths (B)
+    :return: mask tensor containing indices of non-padded part (B, Tmax)
+    :rtype: torch.Tensor
     """
-    batch = int(len(lengths))
+    bs = int(len(lengths))
     maxlen = int(max(lengths))
-    if dim is None:
-        mask = torch.zeros(batch, maxlen)
-    else:
-        dim = int(dim)
-        mask = torch.zeros(batch, maxlen, dim)
+    mask = torch.zeros(bs, maxlen).byte()
```

Comment:
use `torch.zeros(bs, maxlen, dtype=torch.uint8)` because it can byte-allocate once instead of twice as float-allocate ->byte-allocate